assets/js/base/components/cart-checkout/totals/shipping/index.tsx

```diff
@@ -60,15 +60,28 @@ const ShippingAddress = ( {
 	setIsShippingCalculatorOpen,
 	shippingAddress,
 }: ShippingAddressProps ): ReactElement | null => {
+	const hasDefaultCustomerAddress = getSetting(
```

Comment:
I notice that in the cart block, if you have an address (whether you've entered it or not) the `ShippingLocation` component is passed a `shippingAddress`. The check inside that component generates a formatted address and returns if null:

```
if ( ! formattedLocation ) {
		return null;
	}
```

Could we instead move that code to a util and consume in both places?

Reply:
That's a great suggestion. Thank you!  I have updated the code. 👍